config/clusters/2i2c/staging.values.yaml

```diff
@@ -14,6 +14,13 @@ staticWebsite:
   githubAuth:
     enabled: false
 
+deployBinderhubService:
+  enabled: true
```

Comment:
can this instead just be `binderhub-service.enabled`?

Reply:
It would need to be in the binderhub-service schema, but I think that makes sense together with a note that its for charts depending on binderhub-service to toggle installing it.

Reply:
Done in https://github.com/2i2c-org/binderhub-service/pull/39 and updated here